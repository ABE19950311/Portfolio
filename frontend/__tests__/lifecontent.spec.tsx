import '@testing-library/jest-dom/extend-expect'
import '@testing-library/jest-dom'
import { render, screen, cleanup } from '@testing-library/react'
import { renderHook, act } from '@testing-library/react-hooks';
import userEvent from '@testing-library/user-event'
import {getPage} from "next-page-tester"
import Board from "../src/pages/components/board"
import renderer from "react-test-renderer";

describe("Test Lifecontent Component",()=>{

    it("スナップショット",async()=>{
        const {page} = await getPage({
                        route: "/components/lifecontent",
                    })
        const snap = renderer.create(page).toJSON()
        expect(snap).toMatchSnapshot()
    })
})