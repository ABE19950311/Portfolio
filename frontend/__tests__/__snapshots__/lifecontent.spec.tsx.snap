// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Lifecontent Component スナップショット 1`] = `null`;
